python/q617_testing_script.py

```diff
@@ -1,108 +1,14 @@
-    #0, 1, or 2 decimal places. 
-    
-    for i in [0, 1, 2]:
-        loan_amount_lst.append(round(loan_amount, i))
-    #print(loan_amount_lst)
+#This script creates a test file for Q617, using the row_by_row_modification
+#function in utils. The row_by_row_modification_yaml file is used with the function
+#to make modifications to a file specifying rows, columns, and values.
 
-    for i in [0, 1, 2]:
-        property_value_lst.append(round(property_value, i))
-    #print(property_value_lst)
+#The resulting file is placed in the edits_files directory under
+# "../edits_files/Q617_testing/".
 
+#The following code imports the utils package.
+import utils
 
-    #Iterates over 0-4 decimal places for CLTV values. 
-    for i in [0, 1, 2, 3, 4]:
-        #Iterates over each dataframe row to place a property value, 
-        #a loan amount and a calculated CLTV. 
-        for index, row in lar_data.iterrows():
-            row['property_value'] = random.choice(property_value_lst)
-            row['loan_amount'] = random.choice(loan_amount_lst)
-            row['cltv'] = passes_cltv(cltv_rounded = i, row=row)
-
-        #Stores the LAR dataframe as a new LAR dataframe. 
-        new_lar_df = lar_data
-
-        #Stores the number of rows currently in the dataframe.
-        current_row = len(new_lar_df.index) 
-        
-        """Calculates a multiplier taking the ceiling function of 
-        the desired row count over the current row count.""" 
-        multiplier = math.ceil(row_count/current_row)
-        
-        """Concatenates data to produce the number of rows 
-        by the multiplier in a new LAR dataframe."""  
-        new_lar_df = pd.concat([new_lar_df]*int(multiplier))
-        
-        #Drops the number of rows to the count specified. 
-        if (row_count % current_row) != 0:
-            drop_rows = current_row - (row_count % current_row)
-            new_lar_df = new_lar_df[: - (drop_rows)]
-        
-        """Applies the unique_uli function from large_test_files 
-        to the new LAR dataframe to generate a unique set of ULIs."""  
-        new_lar_df = unique_uli(new_lar_df, ts_data['lei'])
-
-        #Modifies TS data for the new number of LAR entries.
-        ts_data["lar_entries"] = len(new_lar_df)
-
-        #Writes a Q617 file with CLTV rounded to i places. 
-        
-        utils.write_file(path="../edits_files/q617_testing/", ts_input=ts_data, 
-            lar_input=new_lar_df, name="test_file_cltv_"+str(i)+"_decimals.txt")
-
-        #Prints a statement indicating that a file has been created with a certain
-        #number of rows and CLTV rounded to i decimal places. 
-        print("Row count is " + str(len(new_lar_df)) + " for cltv rounded to "+str(i)+" decimal places.")
-
-#Stores a filepath for a clean test file, passing syntax and validity. 
+#Stores a filepath for a clean test file. 
 filepath = "../../hmda-platform/data/clean_test_files/bank_0/"
```

Comment:
Do we want to variabalize all or part of this path?
I think at least providing option for bank_0 vs bank_1 would be a good idea.

Maybe as part of the yaml file?